zkmarek/video/slides/ec/animate_subgroup.py

```diff
@@ -19,22 +21,27 @@ def __init__(self, chart: DiscreteEllipticChart, target_color=HIGHLIGHT_COLOR,
         runtime_per_step=0.75):
         self.chart = chart
         self.labels = []
+        self.coordinates = []
         self.runtime_per_step = runtime_per_step
         self.target_color = target_color
         self.duplicates = []
 
     def animate_affine(self, scene: Scene, affine: ECAffine, label: str,
         split_animation=False):
         self.animate_dot(scene, self.chart.find_dot_by_affine(affine),
-            label, split_animation=split_animation)
+            label, affine, split_animation=split_animation)
 
-    def animate_dot(self, scene: Scene, dot: Dot, label: str, split_animation=False):
+    def animate_dot(self, scene: Scene, dot: Dot, label: str, affine: Optional[ECAffine], split_animation=False):
```

Comment:
I would propose to skip affine argument and instead hardcode number in the Slide file.
The "proper" way of exposing such a functionality would be to use `DotOnCurve`, instead of `Dot`, which has affine as a member. Doesn't seem to be worth the trouble, for a single point thou.

Reply:
A note of comment: I feel animate_dot and related is already overcomplicated, but refactoring doesn't seem a good investment.

Reply:
Agree, code moved to slide implementation.